descartes_core/src/planning_graph.cpp

```diff
@@ -995,9 +997,18 @@ double PlanningGraph::linearWeight(JointTrajectoryPt start, JointTrajectoryPt en
     if (start_vector.size() == end_vector.size())
     {
       double vector_diff = 0;
+      double joint_diff = 0;
       for (int i = 0; i < start_vector.size(); i++)
       {
-        vector_diff += fabs(end_vector[i] - start_vector[i]);
+        joint_diff = std::abs(end_vector[i] - start_vector[i]);
```

Comment:
Add an issue to better address large joint changes.  Large joint moves are achievable as long as they happen "far" apart.